examples/zone-printer/README.md

```diff
@@ -32,227 +32,215 @@ that will be used by `kubemci`.
 
 # Tutorial
 
-## 0. Prerequisite
+## Step 0: Before you begin
 
-### 0.1 GCP Project
+1. **GCP Project**
 
-Ensure you that you have a GCP Project. If you don't have one already,
-you can create a project by following the instructions at
-[Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+    Ensure you that you have a GCP Project. If you don't have one already,
```

Comment:
Since you are at it, could you please fix this typo for me?

s/Ensure you that you/Ensure that you/

 This was my fault probably, I am sorry. If you don't have time, that's fine.

Reply:
Thanks.